draft-quic-atsss-reqs.mkd

```diff
@@ -74,49 +77,48 @@ as shown here.
 
 [comment]: # (OB: not sure we need that section, we'll see later)
 
+# Terminology 
+
+This document makes use of 3GPP-specific terms defined in RFC6459, mainly the following ones: 
+
+-    Packet Data Network (PDN): is a packet-based network that
+      either belongs to the operator or is an external network such as
+      the Internet or a corporate intranet.  The user eventually
+      accesses services in one or more PDNs.  The operator's packet core
+      networks are separated from packet data networks either by GGSNs, PDN Gateways (PDN-GWs), or User Plane Function (UPF).
```

Comment:
Do we need to introduce GGSN and PDN-GW, while the ATSSS is restricted to the 5GC?

Reply:
OK to remove them. 

Fixed in: https://github.com/obonaventure/draft-quic-atsss-reqs/pull/11/commits/6f57252ed60a2b5106590ad4cd4dad8e4dff82e8